apps/aeoracle/test/aeoracle_SUITE.erl

```diff
@@ -462,6 +464,38 @@ query_response(Cfg, QueryOpts) ->
 
     {OracleKey, ID, S2}.
 
+query_response_fee_depends_on_response_size(Cfg) ->
+    {OracleKey, ID, S1} = query_oracle(Cfg, #{}, #{}),
+    Trees               = aeo_test_utils:trees(S1),
+    Env                 = aetx_env:tx_env(?ORACLE_RSP_HEIGHT),
+
+    SmallResponse  = <<"small_response">>,
+    BiggerResponse = <<"bigger_oracle_response">>,
+    size(SmallResponse) < size(BiggerResponse),
+
+    %% Deduce minimal fee for response tx with SmallResponse
+    RTx0        = aeo_test_utils:response_tx(OracleKey, ID, SmallResponse, #{}, S1),
+    MinimalFee  = aetx:min_fee(RTx0),
+
+    %% Test oracle response tx with SmallResponse is accepted with MinimalFee
+    RTx1        = aeo_test_utils:response_tx(OracleKey, ID, SmallResponse, #{fee => MinimalFee}, S1),
+    MinimalFee1 = MinimalFee,
+    {ok, Trees} = aetx:check(RTx1, Trees, Env),
+
+    %% Test oracle response tx with BiggerResponse is not accepted with MinimalFee
+    RTx2        = aeo_test_utils:response_tx(OracleKey, ID, BiggerResponse, #{fee => MinimalFee}, S1),
+    MinimalFee2 = aetx:min_fee(RTx2),
+    MinimalFee2 > MinimalFee1,
```

Comment:
Is this meant to be assertion?

Reply:
Yes, ☕️ made me realize it is not going to work :)